### PR TITLE
Added adjustments for 3440x1440px

### PR DESCRIPTION
**Pull request by bsekachev**
### Motivation and context
Adjust UI for different resolutions.

#### Develop (1360x768, 1920x1080, 3440x1440)
Screenshot from 2021-08-03 18-35-09
Screenshot from 2021-08-03 18-35-30
Screenshot from 2021-08-03 18-35-45

#### This branch (1360x768, 1920x1080, 3440x1440)
Screenshot from 2021-08-04 14-43-12
Screenshot from 2021-08-04 14-32-46
Screenshot from 2021-08-04 14-43-50

Also a bit adjusted: 
- Bottom padding on tasks page for different resolutions (used @media screen css)
- Task page preview

From:
Screenshot from 2021-08-03 18-40-45

To:
Screenshot from 2021-08-03 18-41-09

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by bsekachev** [comment]
@ActiveChooN please, look at the patch and let me know if you see any issues here.

File: cvat-ui/src/components/projects-page/project-item.tsx [hunk 1]
```diff
@@ -38,6 +39,26 @@ export default function ProjectItemComponent(props: Props): JSX.Element {
 
     const style: React.CSSProperties = {};
 
+    const NUMBER_OF_RAWS = 3;
+    useEffect(() => {
```
Comment (ActiveChooN) [inline cvat-ui/src/components/projects-page/project-item.tsx:43]: What do you think, can it be helpful to extract this to a separate component or hook to use it for example in cloud storages view?

File: cvat-ui/src/components/projects-page/styles.scss [hunk 2]
```diff
@@ -71,9 +71,20 @@
     justify-content: center;
 }
 
-.cvat-projects-project-item-title,
```
Comment (ActiveChooN) [inline cvat-ui/src/components/projects-page/styles.scss:74]: IMHO, it is a little bit confusing that the title has no cursor pointer

**Review by ActiveChooN — COMMENTED** [review]
Vertical images in project preview are looking stretchy, is this expected?

In your branch:
изображение

In develop:
изображение

File: cvat-ui/src/components/projects-page/styles.scss (hunk 2 above)
Comment (bsekachev) [reply cvat-ui/src/components/projects-page/styles.scss:74]: Fixed

**Comment by bsekachev** [comment]
> Vertical images in project preview are looking stretchy, is this expected?

Fixed

File: cvat-ui/src/components/projects-page/project-item.tsx (hunk 1 above)
Comment (bsekachev) [reply cvat-ui/src/components/projects-page/project-item.tsx:43]: Makes sense

Comment (bsekachev) [reply cvat-ui/src/components/projects-page/project-item.tsx]: Added user hook useCardHeight

**Review by ActiveChooN — APPROVED** [review]

**Pull request merged**